docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -327,255 +388,102 @@ In Geo-Partitioning, the table must be splitted into partitions, and each partit
 
```

Comment:
```suggestion
* Restrict all data in a partition/table/database to one or more defined zones, i.e. keep sensitive data inside a specific set of zones (How about metadata?)
* not allowing some partition/table/database in certain zones, i.e. OK to place in any zone except a set of zones

```

Reply:
I've added this to unresolved questions, with a suggested syntax. Thus, I am going to resolve this comment for now.